tensorrt_llm/models/qwen/convert.py

```
@@ -536,20 +536,25 @@ def convert_hf_qwen(hf_model,
                     v_bias = dup_kv_bias(v_bias, num_key_value_heads,
                                          tensor_parallel)
                 assert (k_weight.shape[0] % (mapping.tp_size * head_size)) == 0
-                assert (v_weight.shape[0] % (mapping.tp_size * head_size)) == 0
-                assert (k_bias.shape[0] % (mapping.tp_size * head_size)) == 0
-                assert (v_bias.shape[0] % (mapping.tp_size * head_size)) == 0
+                assert (v_weight.shape[0] % (mapping.tp_size * head_size)) == 
```

Ruff (0.12.2)

Trailing whitespace

(W291)